### PR TITLE
Bug/metrics restore

### LINKED ISSUE #154: Connection close logging should include user-agent
Connection closing is logged, but fails to include the user-agent, it should.

### PR DESCRIPTION
**Pull request by bbangert**
* Don't cancel a deferred that was already called.
* Restore logging of simplepush successfull/stored delivery based on status.
* Restore updates.handled endpoint timer to track time to deliver.
* Remove logging of connection closing.
* Add changelog for close connection timer.
* Remove metric logging of sendClose timer since its now the default.

Closes #154, #155.

@kitcambridge, @jrconlin r?

### CODE REVIEW

File: autopush/websocket.py [hunk 1]
```diff
@@ -279,11 +279,6 @@ def nukeConnection(self):
         if hasattr(self, "_shutdown_ran"):
             return
 
-        # Uh-oh, we have not been shut-down properly, report detailed data
-        self.ps.metrics.increment("client.error.sendClose_failed",
```
Comment (kitcambridge) [inline autopush/websocket.py]: This might still be helpful, actually...I think we'll only get here if the client didn't respond to the close frame, right?

Comment (bbangert) [inline autopush/websocket.py]: Good point, will re-add it.

**Comment by jrconlin** [comment]
r+

**Comment by kitcambridge** [comment]
r+

**Pull request merged**